packages/frames.js/src/middleware/openframes.test.ts

```diff
@@ -68,8 +68,8 @@ describe("openframes middleware", () => {
         version: "2024-02-09",
       },
       handler: {
-        isValidPayload: (body: JSON) => isXmtpFrameActionPayload(body),
-        getFrameMessage: async (body: JSON) => {
+        isValidPayload: (body: object) => isXmtpFrameActionPayload(body),
```

Comment:
why is the change from `JSON` to `object` necessary?

Reply:
@stephancill I don't understand . As far as I can see you still use `cloneJsonBody()` that parses JSON and returns a json or returns `undefined` in case of an error. Why is it necessary to change the type?

Reply:
I think we should replace `object` with `JsonValue` which is defined in `core/types.ts`.

Reply:
I strongly disagree with you, since you are using Typescript you also want some type checking, otherwise it doesn't make sense to use typescript at all except some code hints. Also state doesn't need to be an object right? It can be just a string, since it is json stringified. 

In order to improve developer experience use can use `zod` to parse state. That improves developer experience, not making types weak.